### PR TITLE
Flav/poke ent subscription

### PR DESCRIPTION
**Pull request by flvndvd**
## Description

In this PR, the upgrade process for workspaces to the Enterprise plan has been refactored. Now, it only requires a valid Stripe subscription Id in conjunction with an Enterprise Plan (identified by a code starting with `ENT_`). It also moves the history behind a call-to-action (CTA), and the search functionality on the table, which was generating console errors.

UpgradeEnterprise

The great form we have to create a new plan will be used in a follow up PR. Stay tuned!

### CODE REVIEW

File: front/components/poke/subscriptions/EnterpriseUpgradeDialog.tsx [hunk 1]
```diff
@@ -0,0 +1,250 @@
...
+        }
+      }
+    };
+    void submit();
+  };
+
+  return (
+    <PokeDialog open={open} onOpenChange={setOpen}>
+      <PokeDialogTrigger asChild>
+        <PokeButton variant="outline" disabled={disabled}>
```
Comment (PopDaph) [inline front/components/poke/subscriptions/EnterpriseUpgradeDialog.tsx:196]: NIT: Could we have the button not full width? 

Super NIT: There's now two different style with Poké and Sparkle buttons on the same modal. Could we only use Poké Buttons since we introduced them?


Super Giga NIT that definitely could be in another PR: Should we migrate all Poké Sparkleq modals to the new PokeDialog?

**Review by PopDaph — APPROVED** [review]
I still think it will make the process a bit more annoying for our Ops since now they have to go to the plans page to create the plan and then come here to set it, but agree it's better to be able to reuse a plan if we want to! 

Thanks also for the global Poké improvements 🙏🏻

File: front/components/poke/subscriptions/EnterpriseUpgradeDialog.tsx (hunk 1 above)
Comment (flvndvd) [reply front/components/poke/subscriptions/EnterpriseUpgradeDialog.tsx:196]: Let me use PokeButton everywhere.

Comment (flvndvd) [reply front/components/poke/subscriptions/EnterpriseUpgradeDialog.tsx:196]: Done
